homeassistant/components/sensorpush_cloud/sensor.py

```diff
@@ -0,0 +1,158 @@
+"""Support for SensorPush Cloud sensors."""
+
+from __future__ import annotations
+
+from typing import Final
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    UnitOfElectricPotential,
+    UnitOfLength,
+    UnitOfPressure,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
```

Comment:
```suggestion
from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
```
those have changed in the meantime (might need a rebase)